rpearce/react-medium-image-zoom
# fix(391): use older browser-friendly selector

## Description

This PR addresses https://github.com/rpearce/react-medium-image-zoom/issues/391

This walks back the usage of `:is()` in a `document.querySelector` in order to support older browsers that don't support that CSS selector.

There is no real penalty for doing one over the other; the `:is()` provided a clearer syntax for writing the selector, and we now dynamically generate it when the module is first evaluated.

## Testing

1. Clone the project
2. Run `npm i`
3. Check out this branch
4. Run `npm start`
5. Test the examples to make sure they work, and ensure there are no error messages in the JS console
